### DISCUSSION TITLE
one more game console Nintendo ique player,multiplayer may join Blueretor fimaly

### DISCUSSION THREAD

**Discussion by hpx1984** [darthcloud/BlueRetro#266] [Q&A] [answered]
Hi Sir Darthcloud

i found a china version N64 (ique Player),  the controller is very like the N64 controller 
now you build many blueretor of games console，i looking your Blueretor family it is very great project for all like TV game console guys,
i think your Blueretor now has one more family member waiting for join ,that is ique player, you can try in future  

now i buy one ESP32D module for try to build one Blueretor, it seems NGC and DC is very simple for build 
hope i can complete it 
 
![image](https://user-images.githubusercontent.com/76941051/166211218-8d51622a-6a4c-4664-bcbd-4f4a80d45764.png)
![image](https://user-images.githubusercontent.com/76941051/166211458-0651c44b-41fe-4941-9afa-67e6d21881ec.png)
![image](https://user-images.githubusercontent.com/76941051/166211511-f30b10e7-a13a-4c4d-b1f1-eec5dba05504.png)

**Comment by darthcloud** [comment, marked as answer]
![405855b6](https://user-images.githubusercontent.com/3744056/166213765-eaf6a199-48b8-490a-b338-19273a4e755d.jpg)
For P2-P4 iQue controller are just regular N64 controller with different plastic. So already supported.

P1 like any portable console got parallel inputs for each buttons and got regular n64 joystick. For this case their isn't enough IO on the ESP32 to support that.

**Comment by hpx1984** [comment]
yes, Sir
i tried to test on ique and it is worked ,
many thanks your commented  

is it Blueretor can working on original Xbox?

**Comment by hpx1984** [comment]
i checked the discussion topic and got to know can't support XBOX
